vsg/vhdlFile/classify/interface_signal_declaration.py

```diff
@@ -3,13 +3,13 @@
 
 from vsg.token import interface_signal_declaration as token
 from vsg.vhdlFile import utils
-from vsg.vhdlFile.classify import expression, identifier_list, mode, subtype_indication
+from vsg.vhdlFile.classify import identifier_list, mode_indication
 
 
 def detect(iToken, lObjects):
     """
     interface_signal_declaration ::=
-        [ signal ] identifier_list : [ mode ] subtype_indication [ bus ] [ := *static*_expression ]
+          signal identifier_list : mode_indication
```

Comment:
I replaced `[ signal ]` with `signal` since the case where signal is absent is handled by interface_unknown_declaration. Unsure which is more correct. Depending on your stance a comment about the change would probably be useful here.